src/DIRAC/WorkloadManagementSystem/Service/SandboxStoreHandler.py

```diff
@@ -236,14 +236,15 @@ def transfer_bulkFromClient(self, fileId, token, _fileSize, fileHelper):
         gLogger.info("Sandbox path will be", sbPath)
         # Register in DB
         credDict = self.getRemoteCredentials()
+        vo = credDict.get("VO", Registry.getVOForGroup(credDict["group"]))
```

Comment:
Is there any point in using `credDict`?

```suggestion
        vo = Registry.getVOForGroup(credDict["group"])
```

(and the other places)

Reply:
technically there's a chance that `"VO"` can be in `credDict`.